.editorconfig

```
@@ -1,3 +1,3 @@
 [*]
 indent_style = tab
-tab_width = 16
+tab_width = 8
```

```suggestion
tab_width = 4
```

---

If we’re gonna decrease it, we could just as well go all the way. #reclaimyourscreenspace

```suggestion
tab_width = 0
```

---

Y'all arguing over a number when there's no less than 14 bytes to be saved here.

```suggestion
```